README.md

```diff
@@ -1,22 +1,3 @@
-# API Project: URL Shortener Microservice for freeCodeCamp
+# URL Shortener Microservice
 
 [![Run on Repl.it](https://repl.it/badge/github/freeCodeCamp/boilerplate-project-urlshortener)](https://repl.it/github/freeCodeCamp/boilerplate-project-urlshortener)
```

Comment:
The bolierplate `README.md` files are eventually only going to be something like the following (per Task number 4 of https://github.com/freeCodeCamp/freeCodeCamp/issues/39721):

# [URL Shortener Microservice](https://www.freecodecamp.org/learn/apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice)

Reply:
Thanks for pointing that out @RandellDawson. Will make the change now.